### DISCUSSION TITLE
Require fill in details at save

### DISCUSSION THREAD

**Discussion by Caseum96** [grafana/grafana#54375] [Dashboards, Panels & Visualizations]
Is it possible to create an option for requiring to fill in the change log after you would save changes to a dashboard?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
